src/libcore/slice.rs

```diff
@@ -655,6 +655,12 @@ impl<'a, T> Default for &'a [T] {
     fn default() -> &'a [T] { &[] }
 }
 
+#[stable(feature = "mut_slice_default", since = "1.5.0")]
+impl<'a, T> Default for &'a mut [T] {
+    #[stable(feature = "mut_slice_default", since = "1.5.0")]
```

Comment:
>You can't experimentally implement part of the trait

Why? While the method signature is (almost) fixed, dynamic behavor still can be unstable.